lib/runner.js

```diff
@@ -44,7 +44,7 @@ runner.run = async () => {
 		return [].concat(
 			result.violations.map(processViolation),
 			result.incomplete.map(processIncomplete)
-		);
+		).reduce((acc, curr) => acc.concat(curr), []);
```

Comment:
I frequently lean on things like lodash to `flatten(arr1, arr2)` as the concat/reduce construction is difficult to read (IMO) and 100% coverage of the reducer function itself is unnecessarily difficult. But I think I'm getting the idea that we're trying to keep external modules to a minimum. Nice that this new code IS covered.

Reply:
I'm not keen to add more dependencies for something so simple, but unless you're familiar then the `.reduce` does take time to get your mind round.
I've removed it in favour of `...spread` which is a lot more readable IMO